tests/integration/quests/test_quest_givers_all_loaded.yaml

```diff
@@ -0,0 +1,244 @@
+# Integration test to verify all quest-giving mobs are loaded at startup
+# This ensures that all quests are accessible to players
+test:
+  id: quest_givers_all_loaded
+  description: "Verify all quest-giving mobs are loaded in the game at startup"
+  author: Integration Test Framework
+  created: 2025-10-18
+  tags: [quest, quest_givers, world_integrity]
+
+setup:
+  character:
+    name: QuestGiverCheck
+    password: test
+    level: 23  # High level wizard for locate command
+    class: warrior
+  start_room: 3001  # Temple area
+  gold: 1000
+
+steps:
```

Comment:
Let's add a new loop: construct so that this can be represented as:
```
loop:
   - variable: mobno
   - values: 3006 3007 ...
   steps:
     - action: command
       command: "locate char <mobno>"
       description: "Verify mob <mobno> is loaded"
       expected:
          - pattern: "\\[.*3.*\\]|room"
          - message: "Should indicate room for mob <mobno>"

This wil seriously shorten this test without making it much worse.